articles/azure/azs-how-create-spn-powershell.md

```diff
@@ -111,10 +110,10 @@ Enter details below to provide values for the variables in the scripts in this a
 
 ### Azure and Azure Stack SPN creation code
 
-<pre><code class="language-PowerShell"># Declare Variables
+<pre><code class="language-PowerShell"># Declare variables
 $AppName = "<output form="appname" name="result" style="display: inline;">TestApp</output>"
 $AppURL = "<output form="appurl" name="result" style="display: inline;">https://test.app</output>"
-$AppPassword = '<output form="apppassword" name="result" style="display: inline;">Password123!</output>'
+$AppPassword = (New-Guid).Guid
```

Comment:
We need to make this a default value but we still need an option to define it in the HTML forms above.

Reply:
Fixed.